src/commands/deploy.ts

```diff
@@ -162,4 +164,23 @@ export default class Deploy extends SfCommand<void> {
     }
     return final;
   }
+  public checkForHookFailures(hookResults: Hook.Result<Deployer[]>): void {
+    if (hookResults.failures?.length) {
+      // display a table of the errors encountered; Plugin Name, Error Message
+      const columns = {
+        pluginName: { header: 'Plugin Name' },
```

Comment:
plugin name might not be that helpful to the user. For example the customer wouldn't know that a failure from plugin-deploy-retrieve means that their metadata deploy failed

Reply:
@mdonnalley This check is only run after the deployers are fetched, way before the user even has an opportunity to pick a deployer from the list.

Reply:
> plugin name might not be that helpful to the user. For example the customer wouldn't know that a failure from plugin-deploy-retrieve means that their metadata deploy failed

I can remove plugin name from the table, leaving error name and message.